extensions/dict_ts29272_avps/dict_ts29272_avps.c

```diff
@@ -2670,16 +2760,16 @@ static int dict_ts29272_avps_load_defs(char * conffile)
 			};
 			CHECK_dict_new( DICT_AVP, &data , NULL, &avp);
 		}
-		/* MDT-Configuration */
+		/* EPS-Subscribed-QoS-Profile */
 		{
 			/* Grouped */
 			struct dict_object * avp;
 			struct dict_avp_data data = {
-				1622,	/* Code */
+				1431,	/* Code */
```

Comment:
why these codes are changed ?

Reply:
They didn't.  The definition file that the code generator used had these 2 AVP's in a different order.  MDT-Configuration is still 1622 and EPS-Subscribed-Qos-Profile is still 1431.  This is identified as a change because of the way the diff tool works.